src/codecs/GroupChatTitleChanged.ts

```diff
@@ -0,0 +1,43 @@
+import { ContentCodec, ContentTypeId, EncodedContent } from '../index'
+
+export const ContentTypeGroupChatTitleChanged: ContentTypeId = {
+  typeId: 'groupChatTitleChanged',
+  authorityId: 'xmtp.org',
+  versionMajor: 1,
+  versionMinor: 0,
+  sameAs(id) {
+    return (
+      this.typeId === id.typeId &&
+      this.authorityId === id.authorityId &&
+      this.versionMajor === id.versionMajor &&
+      this.versionMinor === id.versionMinor
+    )
+  },
+}
+
+export type GroupChatTitleChanged = {
+  // The new title
+  newTitle: string
+
+  // The old title
+  oldTitle: string
```

Comment:
Do we actually use this anywhere?

Reply:
I thought it could be useful for clients, I don't think it's super necessary though. I'll remove it.